conjur-openapi.yml

```diff
@@ -934,6 +934,86 @@ paths:
 
 # ========== Resources ==========
 
+  '/resources':
+    get:
+      tags:
+        - "resources"
+      summary: "Lists resources within an organization account."
+      description: "Lists resources within an organization account.
+
```

Comment:
Why are we using two line spaces between each section?

Reply:
I'm not sure, honestly. This formatting was used in the /resources/{account} route so I followed suit. I can clean up both.